### ISSUE TITLE
Aurelia doesn't work

### ISSUE THREAD

**Issue by brykov** [tastejs/todomvc#1598]
http://todomvc.com/examples/aurelia

Doesn't work for me.

**Comment by dmethvin** [comment]
Some more detail would really help save others time. I checked it out in Chrome and it seems to work fine. In Firefox, Edge,  and IE11, it's getting a 404 on the built file.

**Comment by brykov** [comment]
Sorry, yes, it's FF in my case.

**Comment by CynthiaLYY** [comment]
Thank you for your suggestion. I will write more details about my code and test all functions and objects again. I am a begginer in JS, if you would like to join me to update the js library, it will be very helpful and gateful. PS, my English is not good. Hope I have made myself understood.

**Comment by CynthiaLYY** [comment]
@brykov  Thank you for your attention. I will keep updating and fixing bugs.  I am a begginer in JS, if you would like to join me to update the js library, it will be very helpful and gateful. PS, my English is not good. Hope I have made myself understood.

**Comment by WyohKnott** [comment]
I've got this bug in Firefox 45.0b10:

```
"Potentially unhandled rejection [5] Error loading "github:aurelia/bootstrapper@0.9.3" at http://todomvc.com/examples/aurelia/jspm_packages/github/aurelia/bootstrapper@0.9.3.js
Error loading "dist\bundled" at http://todomvc.com/examples/aurelia/dist\bundled.js
Not Found: http://todomvc.com/examples/aurelia/dist\bundled.js (WARNING: non-Error used)" es6-module-loader.js:7:2593
[4]</</</</k<() es6-module-loader.js:7
f() es6-module-loader.js:7
i() es6-module-loader.js:7
```

**Comment by brykov** [comment]
Obviously, it's related to the missing javascript resource:
```
http://todomvc.com/examples/aurelia/dist\bundled.js
```
Note that backslash before ```bundled.js```
If I change backslash to the usual one like ```http://todomvc.com/examples/aurelia/dist/bundled.js``` the file becomes accessible.

**Comment by arthurvr** [comment]
Works for me, I guess it's fixed by #1545.
